res/css/views/rooms/_GroupLayout.scss

```
@@ -55,77 +55,77 @@ $left-gutter: 65px;
 .mx_MatrixChat_useCompactLayout {
     .mx_EventTile {
         padding-top: 4px;
-    }
 
-    .mx_EventTile.mx_EventTile_info {
-        // same as the padding for non-compact .mx_EventTile.mx_EventTile_info
-        padding-top: 0px;
-        font-size: $font-13px;
-        .mx_EventTile_line, .mx_EventTile_reply {
-            line-height: $font-20px;
-        }
-        .mx_EventTile_avatar {
-            top: 4px;
+        .mx_EventTile.mx_EventTile_info {
```

Hmm, I think this would typically use `&` to join with the outer selector, as in:

```suggestion
        &.mx_EventTile_info {
```

---

Oh yeah, nice save